src/mescal/tapis_de_v3/jobs.clj

```diff
@@ -125,19 +157,26 @@
      (assoc (format-job tapis jobs-enabled? app-info-map job)
             :app-disabled (not (app-enabled? statuses jobs-enabled? app-info))))))
 
+(defn- decode-job-history-description
+  [description]
+  (try+
+    (-> description util/decode-json :message)
+    (catch Exception _
+      description)))
```

Comment:
For some reason, Tapis stores this JSON in the `description` field as a plain string, so we have to decode it to get to the `message`.